modules/etcd-requirements-etcd-hardware-span.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * etcd/etcd-guidance-span.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="requirements-etcd-hardware-span_{context}"]
+= Requirements for etcd, networking, and storage
+
+Consider the following requirements for clusters that span data centers.
+
+== etcd requirements
+
+There is a large list of factors and considerations that go into planning an etcd cluster deployment. When planning an {product-title} cluster that spans data centers, you need to plan for situations that will likely stress or push etcd to the edge of its operational limits.
+
+See _Understanding etcd and the tunables/conditions affecting performance_ for more details on how to maintain operational capabilities and reduce service-affecting events and instability of the cluster.
+
+== Network requirements
+
+The chosen network topology must yield direct IP connectivity between nodes. The minimum effective MTU across the infrastructure should be the maximum MTU used for the deployment. Using a lower MTU is OK.
+
+For more information, see _Understanding and Validating MTU setting with {product-title} 4.x_. The latency needs are ultimately defined by the services by using the network. See the sections related to etcd and storage for more details on requirements.
+
+In addition to the base networking requirements, you need to think about how applications will be accessed. Some top level Global Service Load Balancing (GSLB) method will be needed outside of {product-title} to enable external traffic to connect to the {product-title} control plane services and ingress controllers.
+
+== Storage requirements
+
+When considering a cluster deployment that spans data centers, special consideration needs to be given to the selected storage integration to ensure that it also meets multisite requirements, as it pertains to accessibility from all sites, fault tolerance, high availability, and so on.
+
+An "object storage" solution should be used for the registry, and this storage solution needs to be in addition to any PV storage integration used for application volumes or workloads. This "object storage" solution should also have the same special considerations given to accessibility from all sites, fault tolerance, high availability, and so on.
+
+Since Disk IO is a critical factor in the health of etcd database, it is required that they are deployed on a high speed, low latency media. See etcd guidance on _etcd peer round trip time_ and _etcd database size_ for more details on the exact requirements to meet.
```

Comment:
Couple things, per [ISG](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#s): Do not use (since) as a synonym for because. Also, I think it should be _disk I/O_. I do not see _Disk IO_ elsewhere in the docs. An AI web search suggests it is singular. 

```suggestion
Because disk I/O is a critical factor in the health of etcd database, it is required that to deploy disk I/O on a high speed, low latency media. See etcd guidance on _etcd peer round trip time_ and _etcd database size_ for more details on the exact requirements to meet.
```